templates/firefox-ios.xliff

```diff
@@ -3553,7 +3553,7 @@ Private Tab</source>
         <note>On top of the bottom toolbar, there can be a microsurvey prompt, this is the title for the button that appears on the prompt that allows the user to tap on and navigates them to the microsurvey to respond to.</note>
       </trans-unit>
       <trans-unit id="Microsurvey.Prompt.Close.AccessibilityLabel.v127" xml:space="preserve">
-        <source>Close Survey Prompt</source>
+        <source>Close</source>
```

Comment:
You will need to udpate the string ID as the string changes here (more info about that [here](https://mozilla-l10n.github.io/documentation/localization/dev_best_practices.html#string-identifiers-ids))